news/_posts/2020-05-13-the-new-order-pages-in-prestaShop-1.7.7.md

```diff
@@ -0,0 +1,225 @@
+- orders waiting for a merchant action following an accepted payment in light blue
+- when no action is needed from the merchant as when the product is delivered in green
+- special status as payment error order in red or black
+
+The merchants have a concise view of the work remaining to be done.
+
+***How it works***: 
+
+The statuses colour is set by default. They are still configurable in the Order Setting page.
+
+
+- **Link to the customer's profile**: Creates easy access to the customer's profile by clicking on its name.
+
+![Right click on the customer's link](/assets/images/2020/05/link-customer.png "Right click on the customer's link")
+
+***What is the goal***: 
+
+Before the merchant used to have to open the order to get access to the customer page. Now, the merchant can do it from the orders listing.
+
+***How it works***:
+
+Clicking on the customer name opens the customer page in a new tab.
+
+
+
+
+- **Bulk action redesign** : The bulk actions' design and its position were updated to make it more visible and quicker to use.
+
+![Change_order's statuses](/assets/images/2020/05/choose_an_order.png "Change_order's statuses")
+
+_Updates the last 4 orders into payment accepted_
+
+***What is the goal***: 
+
+To help the merchants to process multiple orders in 3 clicks.
+
+***How it works***: 
+
+Select several orders, choose the new order status in the pop-in then validate it to update the status on the orders.
+
+
+
+
+- **Open in new tabs**: Open commands in new tabs by mass action to process them as the merchant proceeds.
+
+![Open in new tabs](/assets/images/2020/05/open_in_new_tabs.png "Open in new tabs")
+
+_Opens the last 4 orders in new tabs_
+
+***What is the goal***: 
+
+It helps the merchants to open multiple orders in 3 clicks.
+
+***How it works***: 
+
+Choose several orders, select _open in new tabs option_ on bulk action's option to open them in new tabs.
+
+## II. **For the order details, we have added the following features :**
+
+**Even if we keep the objective of saving time and productivity, the improvements and features are focused on readability and the position of the information. Indeed, some important information did not have the display corresponding to its importance. For example, the list of products that were at the bottom of the page is now at the top of the page as the first information to be consulted.**
+
+**Thus with better readability, we allow the merchant to be more efficient in his work.**
+
+
+- **Pagination on the product list**: Allows to navigate in the order product list through the pagination.
+
+![Order's_pagination](/assets/images/2020/05/order_pagination.png "Order's pagination")
+
+***What is the goal***:
+
+Avoid an endless and unreadable list of products. Pagination provides structure in the products&#39; navigation. Also, the list has been moved to the top of the page to avoid the merchant from scrolling to the bottom of the page to see the products ordered.
```

Comment:
```suggestion
Avoid an endless and unreadable list of products. Pagination provides structure in the products&#39; navigation. Also, the list has been moved to the top of the page to avoid the merchant from scrolling to the bottom of the page.
```